cinch/group_vars/rhel8

```diff
@@ -0,0 +1,41 @@
+python_pip_package: python3-pip
+
+all_repositories:
+  baseos:
+    name: rhel8-baseos
```

Comment:
I guess you have specific repos that you use, I suggest to stick with same format we have in rhel 7.